generated/nidaqmx/system/physical_channel.py

```diff
@@ -1180,33 +1180,62 @@ def teds_version_num(self):
 
         return val.value
 
-    def clear_teds(self):
+
+    @property
+    @deprecation.deprecated(deprecated_in="0.6.6", details="Use ai_supported_meas_types instead.")
+    def ai_meas_types(self):
+        return self.ai_supported_meas_types
+
+    @property
+    @deprecation.deprecated(deprecated_in="0.6.6", details="Use ao_supported_output_types instead.")
+    def ao_output_types(self):
+        return self.ao_supported_output_types
+
+    @property
+    @deprecation.deprecated(deprecated_in="0.6.6", details="Use ci_supported_meas_types instead.")
+    def ci_meas_types(self):
+        return self.ci_supported_meas_types
+
+    @property
+    @deprecation.deprecated(deprecated_in="0.6.6", details="Use co_supported_output_types instead.")
+    def co_output_types(self):
+        return self.co_supported_output_types
+
+    def clear_teds(self, physical_channel):
         """
         Removes TEDS information from the physical channel you specify.
         This function temporarily overrides any TEDS configuration for
         the physical channel that you performed in MAX.
+
+        Args:
+            physical_channel (str): 
         """
         cfunc = lib_importer.windll.DAQmxClearTEDS
         if cfunc.argtypes is None:
             with cfunc.arglock:
                 if cfunc.argtypes is None:
                     cfunc.argtypes = [
-                        ctypes_byte_str]
+                        ctypes_byte_str, ctypes_byte_str]
```

Comment:
this isn't right

```c
EXTERNC int32 __CFUNC DAQmxClearTEDS(const char physicalChannel[])
```

Reply:
Looks like a few known issues like this you are resolving.

Reply:
Resolved.